scripts/ci-footprint-and-performance/tests/001-wait-for-greenboot.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+retries=15
```

Comment:
Any reason for 7.5m wait? Let's make it 10m at least?

Reply:
Idk, I just copied it from somewhere else :) Changed